src/main/java/org/mvel2/asm/Opcodes.java

```diff
@@ -58,6 +58,7 @@
     int V1_6 = 0 << 16 | 50;
     int V1_7 = 0 << 16 | 51;
     int V1_8 = 0 << 16 | 52;
+    int V1_9 = 0 << 16 | 53;
```

Comment:
I just tried to do the minimal change to make it work :) But I agree upgrading ASM would be probably better. On the other that seems like quite a lot of work considering the package names are different.